config/initializers/sufia.rb

```diff
@@ -208,7 +208,8 @@
      "2014-06" => "2014-06",
      "2014-11" => "2014-11",
      "2015-06" => "2015-06",
-     "2015-11" => "2015-11"
+     "2015-11" => "2015-11",
+     "2016-06" => "2016-06"
```

Comment:
Then it probably doesn't belong in this pull request.  Or at least not the same commit.